README.md

```diff
@@ -75,6 +75,36 @@ seneca()
   .act('color:red')
 ```
 
+Example with HTTPS
+
+To enable HTTPS make sure to set the protocol option to 'https' and in the listen function set the serverOptions object with the keyPemPath and the certPemPath keys with string paths leading to the SSL's pem key & certificate files
+
+```js
+var seneca = require('seneca')
+
+seneca()
+  .use(color)
+  .listen(
+    type: 'http',
+    port: '8000',
+    host: 'localhost',
+    protocol: 'https',
+    serverOptions : {
+      keyPemPath : './key.pem',
```

Comment:
we should make `serverOptions` more flexible and just pass in the object to `Https.createServer`.  This will allow people to pass in a key/cert config as well as pfx option. This also makes it so seneca isn't responsible for loading a pem file.